AdminDashboard/Exceptions/AccountNotFoundException.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace AdminDashboard.Exceptions
+{
+    public class AccountNotFoundException : Exception
+    {
+        public AccountNotFoundException() { }
+    }
+}
```

Comment:
I haven't done much with C# - is there a way of grouping all definitions of exceptions in one file? Seems a bit weird to have so many small files around, but then again they're one class per file, so it might be all good.